src/ListView/features/CreatePage/CreatePageForm.jsx

```diff
@@ -0,0 +1,125 @@
+import React, { useEffect, useState } from "react";
+import { colorTextErrorOnLight } from "@sailthru/design-tokens";
+import {
+  FormGroup,
+  Radio,
+  RadioGroup,
+  Subtext,
+} from "@sailthru/stui-components";
+import { Fieldset, Label, Input } from "@sailthru/stui-elements";
+
+function CreatePageForm({
+  name = "",
+  onChange,
+  isDuplicateName,
+  isNameChanged,
+}) {
+  const [namingError, setNamingError] = useState(false);
+  const [optoutNamingError, setOptoutNamingError] = useState(false);
+
+  // function to check for empty, alphanumeric, and 'oc' page names
+  const handleNamingError = function () {
+    setNamingError("");
+    const alphanumericRegex = /^\d*[a-zA-Z0-9][a-zA-Z0-9]*$/;
+    const allowDashInName = name.replace(new RegExp("-", "g"), "");
+    if (name.length < 1) {
+      setNamingError("Please enter a page name.");
+      return;
+    }
+    if (!allowDashInName.match(alphanumericRegex)) {
+      setNamingError("Page names must be alphanumeric.");
+      return;
+    }
+    if (name === "oc") {
+      setNamingError(
+        "The page name 'oc' is a reserved page name. Please select a new page name."
+      );
+      return;
+    }
+    if (name == "optout") {
```

Comment:
nit-pick: I think this should be a `===`
```suggestion
    if (name === "optout") {
```

Reply:
updated